src/cts/src/TritonCTS.cpp

```diff
@@ -1262,6 +1265,7 @@ Clock TritonCTS::forkRegisterClockNetwork(
   odb::dbMaster* master = db_->findMaster(options_->getRootBuffer().c_str());
   topBufferName = "clkbuf_regs_" + std::to_string(regTreeRootBufIndex_++) + "_"
                   + clockNet.getSdcName();
+  options_->recordBuffer(master);
```

Comment:
It would be safer to use a db observer to capture instance creation rather the annotation each call site.

Reply:
good point. Might complicate the dummy load marking, but I'll take a look